app/services/hashtag_autocomplete_service.rb

```diff
@@ -159,8 +164,9 @@ def lookup(slugs, types_in_priority_order)
   #                        bother searching subsequent types if the first types in
   #                        the array already reach the limit.
   # @returns {Array} The results as HashtagItems
-  def search(term, types_in_priority_order, limit: 5)
+  def search(term, types_in_priority_order, limit: nil)
     raise Discourse::InvalidParameters.new(:order) if !types_in_priority_order.is_a?(Array)
+    limit = SiteSetting.experimental_hashtag_search_result_limit if limit.nil?
```

Comment:
Shouldn't we do:

```
def search(term, types_in_priority_order, limit: SiteSetting.experimental_hashtag_search_result_limit)
```

Reply:
I wasn't sure we could do that, I should have just tried it :sweat_smile: No problem I will fix